src/components/BooksForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
BookForm as well is a container component as it will be managing a state for creating books and should go into the `container` folder.